README.md

```diff
@@ -125,13 +130,13 @@ Deploy Azure IoT Hub and configure service communication to Azure Storage for Fi
 > [!TIP]
 > If you don't have custom DNS and SSL setup completed, use a dummy value when calling the script.
 
-* Run the following script to deploy and configure Azure IoT Hub and Storage. The names of the resources are identical to the first script and composed by the environment variables loaded upfront.
+1. Run the following script to deploy and configure Azure IoT Hub and Storage. The names of the resources are identical to the first script and composed by the environment variables loaded upfront.
 
 ```bash
 ./deploy/quickstart-add-iothub.sh "<your_custom_dns>"
 ```
```

Comment:
~~~suggestion
1. Run the following script to deploy and configure Azure IoT Hub and Storage.

   The names of the resources are identical to the first script and composed by the environment variables loaded upfront.

   ```bash
   ./deploy/quickstart-add-iothub.sh "<your_custom_dns>"
   ```
~~~